tacker/sfc/plugin.py

```diff
@@ -196,7 +196,9 @@ def update_sfc(self, context, sfc_id, sfc):
 
 class NeutronClient:
     def __init__(self):
-        auth_url = cfg.CONF.keystone_authtoken.auth_uri + '/v2.0'
+        auth_url = cfg.CONF.keystone_authtoken.auth_uri
+        if re.match('.+(v3/?|v2\.0/?)$', auth_url) is not None:
```

Comment:
Another thing it would be better to do the following (from my perspective):

```
import urlparse

if re.match('.+(v3/?|v2\.0/?)$', auth_url) is None:
    auth_url = urllib.parse.urljoin(auth_url, '/v2.0')
    # or  auth_url = '{0}/v2.0'.format(.rstrip('/'))
```

If a user specified the `auth_url` as `http://172.16.0.3:5000/` this would result in `http://172.16.0.3:5000//v2.0` (I don't know if this is an problem for KeyStone)